types.d.ts

```diff
@@ -31,16 +31,25 @@ declare module "@tableland/sqlparser" {
    * The WASM `Exports` object cache.
    */
   export const __wasm: WebAssembly.Exports | undefined;
+
+  export type NormalizeResult = sqlparser.NormalizeResult;
 }
 
 declare namespace sqlparser {
   /**
-     * Validate and normalize a string containing (possibly multiple) SQL statement(s).
+   * Information about a (set of) normalized SQL statement(s).
```

Comment:
this was a tricky setup, because we have this global namespace to deal with, but I wanted the type to be exported from the actual module itself